ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java

```diff
@@ -523,282 +316,193 @@ private void loadProperties() {
-        return this.proxy;
-    }
-
-    void setForce(final boolean force) {
-        this.force = force;
+    public String getHelp() {
+        return cmd.getOptionValue(COMMAND_HELP.getLongOpt());
     }
-
-    public boolean isForce() {
-        return this.force;
-    }
-
-    void setRetries(final int retries) {
-        this.retries = retries;
+    public ViewType getOutputFormat() {
+        return ViewType.valueOf(this.getOptionValueWithDefault(VIEW_TYPE.getLongOpt(), ViewType.CLI).toString().toUpperCase());
     }
 
+    public boolean isCertificateVerification() { return !this.optionExists(INSECURE.getLongOpt()); }
+    public boolean isHttps() {  return !this.optionExists(HTTP.getLongOpt()); }
+    public boolean isHelp() { return this.help; }
+    void setHelp(final boolean help) { this.help = help; }
+    public Level getConsoleLogLevel() { return this.consoleLogLevel; }
+    void setConsoleLogLevel(final Level console) {this.consoleLogLevel = console; }
+    public Level getFileLogLevel() { return this.fileLogLevel; }
+    void setFileLogLevel(final Level file) {this.fileLogLevel = file; }
     public int getRetries() {
-        return this.retries;
-    }
-
-    public String getPrefix() {
-        return this.prefix;
-    }
-
-    void setPrefix(final String prefix) {
-        this.prefix = prefix;
-    }
-
-    public boolean isChecksum() {
-        return this.checksum;
-    }
-
-    void setChecksum(final boolean checksum) {
-        this.checksum = checksum;
+        final String retryString = this.getOptionValueWithDefault(RETRIES.getOpt(), DEFAULT_RETRIES).toString();
+        try {
+            return Integer.parseInt(retryString);
+        } catch (final NumberFormatException e) {
+            System.err.printf("Error: Argument (%s) to '-r' was not a number\n", retryString);
+            System.exit(1);
+        }
+        return -1;
     }
 
+    // convenience getters for public options
+    public String getBucket() { return this.getOptionValue(BUCKET.getOpt()); }
+    public String getDirectory() { return this.getOptionValue(DIRECTORY.getOpt()); }
+    public String getObjectName()  { return this.getOptionValue(OBJECT_NAME.getOpt()); }
+    public boolean isForce() { return this.optionExists(FORCE.getLongOpt()); }
+    public String getPrefix()  { return this.getOptionValueWithDefault(PREFIX.getOpt(), "").toString(); }
+    public boolean isChecksum() { return this.optionExists(CHECKSUM.getLongOpt()); }
     public Priority getPriority() {
-        return this.priority;
-    }
-
-    void setPriority(final Priority priority) {
-        this.priority = priority;
+        final String priorityString = this.getOptionValue(PRIORITY.getLongOpt());
+        try {
+            if (priorityString == null) {
+                return null;
+            } else {
+                return Priority.valueOf(priorityString.toUpperCase());
+            }
+        } catch (final IllegalArgumentException e) {
```

Comment:
This should follow the normal exception handling path and let the caller handle the printing of the exception.